Jenkinsfile

```diff
@@ -44,7 +44,7 @@ for(i = 0; i < buildTypes.size(); i++) {
                             // The -Dmaven.repo.local=${pwd()}/.repository means that Maven will create a
                             // .repository directory at the root of the build (which it gets from the
                             // pwd() Workflow call) and use that for the local Maven repository.
-                            def mvnCmd = "mvn -Pdebug -U clean install ${runTests ? '-Dmaven.test.failure.ignore=true' : '-DskipTests'} -V -B -Dmaven.repo.local=${pwd()}/.repository" 
+                            def mvnCmd = "mvn -Pdebug -U clean install javadoc:javadoc ${runTests ? '-Dmaven.test.failure.ignore=true' : '-DskipTests'} -V -B -Dmaven.repo.local=${pwd()}/.repository" 
```

Comment:
PR builds are building pr-head rather than pr-merge, so we may accidentally merge something that breaks the CI build. So we need to be vigilant about that.